bundles/org.openhab.core.config.core/src/main/java/org/openhab/core/config/core/internal/validation/OptionsValidator.java

```diff
@@ -31,12 +34,22 @@
             return null;
         }
 
-        // Option values are a string, so we can do a simple compare
-        if (param.getOptions().stream().map(o -> o.getValue()).noneMatch(v -> v.equals(value.toString()))) {
+        boolean invalid;
+
+        if (param.getType() == ConfigDescriptionParameter.Type.DECIMAL) {
```

Comment:
I am afraid checking for `DECIMAL` is not enough. What if a parameter has type `INTEGER`?

Reply:
Originally I included `integer` but after thinking about it, I thought it‘s not necessary, because they are stored as `1` and not as `1.0` but I‘ll check.